virttest/utils_net.py

```diff
@@ -4351,3 +4351,46 @@ def delete_ovs_bridge(ovs_bridge_name, session=None, ignore_status=False):
     return utils_misc.cmd_status_output(tmux_cmd, shell=True, verbose=True,
                                         ignore_status=ignore_status,
                                         session=session)
+
+
+def create_linux_bridge(linux_bridge_name, iface_name, ignore_status=False):
+    """
+    Create linux bridge and connect a physical interface to the bridge via tmux command on local host
+    1. use tmux to avoid the network broken;
+    2. add a physical interface to the bridge, and set the bridge up;
+
+    :param linux_bridge_name: The linux bridge
+    :param iface_name: the physical interface to be attached to the bridge
+    :param ignore_status: Whether to raise an exception when command fails
+    :return: bridge created or raise exception
+    """
+    # Create bridge
+    if not utils_package.package_install(['tmux', 'dhcp-client']):
+        raise exceptions.TestError("Failed to install the required packages.")
+    current_iface = process.run("ls /sys/class/net", shell=True, verbose=True, ignore_status=True).stdout_text.split()
+    if linux_bridge_name in current_iface:
+        raise exceptions.TestError("Create bridge error as there is already an interface named '%s' on the host"
+                                   % linux_bridge_name)
+    cmd = 'tmux -c "ip link add name {0} type bridge; ip link set {1} up;' \
+          ' ip link set {1} master {0}; ip link set {0} up;' \
+          ' pkill dhclient; sleep 6; dhclient {0}; ifconfig {1} 0"'.format(linux_bridge_name, iface_name)
+    return utils_misc.cmd_status_output(cmd, shell=True, verbose=True,
+                                        ignore_status=ignore_status)
+
+
+def delete_linux_bridge(linux_bridge_name, iface_name, ignore_status=False):
+    """
+    Delete the linux bridge on the host, and recover the network on the physical interface
+
+    :param linux_bridge_name: The linux bridge
+    :param iface_name: the physical interface to be attached to the bridge
+    :param ignore_status: Whether to raise an exception when command fails
+    :return: bridge deleted or raise exception
+    """
+    # Delete the linux bridge
+    if not utils_package.package_install(['tmux', 'dhcp-client', 'procps-ng']):
+        raise exceptions.TestError("Failed to install the required packages.")
+    cmd = 'tmux -c "ip link set {1} nomaster; ip link delete {0}; pkill dhclient; ' \
+          'sleep 5; dhclient {1}"'.format(linux_bridge_name, iface_name)
+    return utils_misc.cmd_status_output(cmd, shell=True, verbose=True,
```

Comment:
Better return True or False ,rather than standard output

Reply:
Thank you for the suggestion.
update to:
s, o = utils_misc.cmd_status_output(cmd, shell=True, verbose=True, ignore_status=ignore_status)
return s

Reply:
> Better return True or False ,rather than standard output
As the utils_misc.cmd_status_output() with return "status" and "outputs", if we only return True or False, it will loss some information in the outputs like error message if create failure.